pkg/dashboard/server/api.go

```diff
@@ -480,6 +480,14 @@ func deleteEnvFromDB(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	fluxUser := "flux-" + envNameToDelete
```

Comment:
Can you make this a string pattern and a constant.

This pattern is used for the creation as well.